addons/survey/static/src/js/survey_renderers.js

```diff
@@ -1,55 +1,27 @@
 /** @odoo-module */
 
-import Dialog from 'web.Dialog';
```

Comment:
(Small note, there is a strange gap in your commit message when displayed on github right after "specs", could be due to some undesired automatic formatting?)

![image](https://user-images.githubusercontent.com/41282536/164254160-e351f3a1-ec33-4aa7-ba6c-ce6f3043cfda.png)